wiki/Tournaments/3WC/2025/en.md

```diff
@@ -90,28 +90,54 @@ The 3 Digit World Cup 2025 is run by various community members.
+| South Korea ::{ flag=KR }:: | ::{ flag=GB }:: United Kingdom | Feb 22 (Sat) 13:30 UTC | [3wc_osu](https://twitch.tv/3wc_osu) | [^winners-bracket] |
+| Austria-Hungary ![Austria-Hungary](img/AH.png) | ::{ flag=FI }:: Finland | Feb 22 (Sat) 16:00 UTC | [3wc_osu](https://twitch.tv/3wc_osu) | [^losers-bracket] |
+| France ::{ flag=FR }:: | ::{ flag=BR }:: Brazil | Feb 22 (Sat) 17:00 UTC | [3wc_osu](https://twitch.tv/3wc_osu) | [^losers-bracket] |
+| Canada ::{ flag=CA }:: | ::{ flag=CL }:: Chile | Feb 22 (Sat) 22:00 UTC | [3wc_osu](https://twitch.tv/3wc_osu) | [^losers-bracket] |
 
-### Sunday, February 16
+### Sunday, February 23
 
 | Red Team | Blue Team | Match time | Twitch stream |  |
 | --: | :-- | :-- | :-: | :-: |
-| United Kingdom ::{ flag=GB }:: | ![Oceania](img/OC.png) Oceania | Feb 16 (Sun) 10:00 UTC | [3wc_osu](https://twitch.tv/3wc_osu) | [^winners-bracket] |
-| South Korea ::{ flag=KR }:: | ::{ flag=GR }:: Greece | Feb 16 (Sun) 14:00 UTC | [3wc_osu](https://twitch.tv/3wc_osu) | [^winners-bracket] |
-| Brazil ::{ flag=BR }:: | ::{ flag=RU }:: Russian Federation | Feb 16 (Sun) 16:00 UTC | [3wc_osu](https://twitch.tv/3wc_osu) | [^winners-bracket] |
-| Poland ::{ flag=PL }:: | ::{ flag=CA }:: Canada | Feb 16 (Sun) 20:00 UTC | [3wc_osu](https://twitch.tv/3wc_osu) | [^winners-bracket] |
-| United States ::{ flag=US }:: | ![Austria-Hungary](img/AH.png) Austria-Hungary | Feb 16 (Sun) 20:00 UTC | [3wc_osub](https://twitch.tv/3wc_osub) | [^winners-bracket] |
+| United States ::{ flag=US }:: | ::{ flag=PH }:: Philippines | Feb 23 (Sun) 04:00 UTC | [3wc_osu](https://twitch.tv/3wc_osu) | [^winners-bracket] |
+| Germany ::{ flag=DE }:: | ::{ flag=RU }:: Russian Federation | Feb 23 (Sun) 17:00 UTC | [3wc_osu](https://twitch.tv/3wc_osu) | [^winners-bracket] |
+| Poland ::{ flag=PL }:: | ::{ flag=RO }:: Romania | Feb 23 (Sun) 18:00 UTC | [3wc_osub](https://twitch.tv/3wc_osub) | [^winners-bracket] |
 
 ## Mappools
 
+### Quarterfinals
+
+[View the showcase VOD here](https://www.youtube.com/watch?v=2DQ70wI-MOw)
+
+- No Mod
+  1. [Caramell - Caramelldansen (Ryu\* Remix) (AJT) \[(\* V\* )\]](https://osu.ppy.sh/beatmapsets/1770758#osu/3624869)
+  2. [Hideous Divinity - The Deaden Room (Orkay) \[Precognition of Blood\]](https://osu.ppy.sh/beatmapsets/2325281#osu/4984190)
+  3. [Kakeru - Do you love me? feat. Rntl (lcfc) \[kurukuru\]](https://osu.ppy.sh/beatmapsets/1235834#osu/2569027)
+  4. [Feryquitous vs. xi - Orca (lcfc) \[Succulent\]](https://osu.ppy.sh/beatmapsets/1166748#osu/2433641)
+- Hidden
+  1. [ASIAN KUNG-FU GENERATION - After Dark (-NeBu-) \[Riding On Faint Hopes Carried By The Night Wind\]](https://osu.ppy.sh/beatmapsets/2268520#osu/4831196)
+  2. [POLKADOT STINGRAY - Keshin (Keqing) \[Atalanta's Extreme\]](https://osu.ppy.sh/beatmapsets/1311266#osu/2987479)
+- Hard Rock
+  1. [Ice - Amber Wishes (Deemo edit) (Astronic) \[_Epreus' Diminuendo\]](https://osu.ppy.sh/beatmapsets/1610241#osu/4254657)
```

Comment:
```suggestion
  1. [Ice - Amber Wishes (Deemo edit) (Astronic) \[\_Epreus' Diminuendo\]](https://osu.ppy.sh/beatmapsets/1610241#osu/4254657)
```

Reply:
technically none of these escapes are needed (pairs of square brackets, single underscores)